### PR TITLE
Chore: Tidy Javascripts

### PR DESCRIPTION
**Pull request by naseberry**
#### What
Tidying of JS aspects of app

#### Why
To make `package.json` valid and to remove redundant Js modules.

#### How
- Validate package.json by adding in the required key value pairs
- Remove redundant js modules that were required but were fully commented

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+    "type": "git",
+    "url": "git+https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence.git"
+  },
+  "author": "LAA Developers",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/issues"
+  },
+  "homepage": "https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence#readme",
   "dependencies": {
```
Comment (mathan11480-moj) [inline package.json:16]: Can you also add "main:application.js" for completion?

Comment (naseberry) [reply package.json:16]: thanks @mathan11480-moj but we don't utilise an entry file in this app.

Comment (mathan11480-moj) [reply package.json:16]: 👍 do we even ship this as an npm package?

Comment (naseberry) [reply package.json:16]: no, we do not @mathan11480-moj

**Review by jsugarman — APPROVED** [review]
👍 assuming this has been checked on an environment, this is great!

**Pull request merged**